client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -4,21 +4,20 @@ import paymentsImage from 'calypso/assets/images/illustrations/payments.svg';
 import PurchaseDetail from 'calypso/components/purchase-detail';
 
 export default localize( ( { isJetpack, translate } ) => {
-	const supportDocLink = localizeUrl(
-		isJetpack
-			? 'https://jetpack.com/support/pay-with-paypal/'
-			: 'https://wordpress.com/support/pay-with-paypal/'
-	);
 	return (
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				buttonText={ translate( 'Collect PayPal payments' ) }
 				description={ translate(
 					'Add a button to any post or page to collect PayPal payments for physical products, services, or donations.'
 				) }
-				href={ supportDocLink }
+				{ ...( isJetpack
+					? {
+							href: localizeUrl( 'https://jetpack.com/support/pay-with-paypal/' ),
+							target: '_blank',
+					  }
+					: { supportContext: 'sell-online-paypal' } ) }
```

Comment:
I can imagine that we're going to need to reuse this logic in a ton of places. Do you think it's worth it to extract it to a small helper so that we can get this functionality out of the box for any link we create?